src/mobile/pages/pages.stache

```diff
@@ -84,7 +84,7 @@
   {{#if(currentPage.canPreview)}}
     <p>You have documents that can be previewed.</p>
     <button
-      class="btn btn-default btn-navigate open-preview {{#is(button, focusedButton)}}{{focusedButtonRendering()}}{{/is}}"
```

Comment:
when a visited page is selected from the navigation panel, this is how the last-used-button gets re-focused

needs to be restored or refactored

Reply:
This is only for the preview button, which isn’t part of the navigation panel flow.

I had copied this from the button above when I first added this button, and it was only after that I realized this was actually causing an issue. Because `button` and `focusedButton` are both `undefined`, the preview button was _always_ showing up as focused when the bubble first appeared.

I still believe this new change is correct, but please let me know if I’m missing something.

Reply:
You are totally correct, I mistook this for the original code and didn't notice this was from a previous copy paste

lgtm!